workers/unity/Packages/com.improbable.gdk.core/Utility/ViewCommandBuffer.cs

```diff
@@ -12,13 +15,29 @@ namespace Improbable.Gdk.Core
     /// </summary>
     public class ViewCommandBuffer
     {
-        private readonly Queue<BufferedCommand> bufferedCommands = new Queue<BufferedCommand>();
-
-        private const string LoggerName = "ViewCommandBuffer";
+        private const string LoggerName = nameof(ViewCommandBuffer);
 
         private const string UnknownErrorEncountered =
             "ViewCommandBuffer encountered unknown command type during buffer flush.";
 
+        private readonly EntityManager EntityManager;
+        private readonly ILogDispatcher LogDispatcher;
+        private readonly Action<Entity, ComponentType, object> setComponentObjectAction;
+        private readonly Queue<BufferedCommand> bufferedCommands = new Queue<BufferedCommand>();
+
+        private static readonly MethodInfo setComponentObjectMethodInfo =
+            typeof(EntityManager).GetMethod("SetComponentObject", BindingFlags.Instance | BindingFlags.NonPublic, null,
+                new Type[] { typeof(Entity), typeof(ComponentType), typeof(object) },
+                new ParameterModifier[] { });
```

Comment:
Add a TODO to remove this once we merge with master as we will have the extension methods

Reply:
will we need this in multiple places? if not, I would prefer to keep it isolated in this place and get rid of the extension methiods

Reply:
I think its used in interfacing with Monobehaviours mostly - not sure how many places its being used